docs/models/models.md

```diff
@@ -0,0 +1,146 @@
+<a name="collections"></a>
```

Comment:
This link is unnecessary -the following header will do it anyway

Reply:
ok. Fixed in ef4582b